Bottom: 4d988aa7d0ed2fa309ab4a5b3d3a7c62b58ea245
Top:    fc8c791edbf69034d4c4a7f70687b65af81d5897
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2017-12-12 10:47:07 -0600

DO-NOT-SUBMIT! Local testing-related changes

---
diff --git a/docker/buildbot-worker-ntel/buildbot/.gitconfig b/docker/buildbot-worker-ntel/buildbot/.gitconfig
index efd988f..de7ba0c 100644
--- a/docker/buildbot-worker-ntel/buildbot/.gitconfig
+++ b/docker/buildbot-worker-ntel/buildbot/.gitconfig
@@ -1,3 +1,3 @@
 [user]
-	name = BuildBot
-	email = buildbot@novatechweb.com
+	name = Andrew Cooper
+	email = andrew.cooper@novatechweb.com
diff --git a/docker/buildbot-worker-ntel/buildbot/.ssh/config b/docker/buildbot-worker-ntel/buildbot/.ssh/config
new file mode 100644
index 0000000..3d86e2c
--- /dev/null
+++ b/docker/buildbot-worker-ntel/buildbot/.ssh/config
@@ -0,0 +1,4 @@
+Host git.andrew.cooper
+    HostName git.novatech-llc.com
+    IdentityFile ~/.ssh/id.andrew.cooper.me
+    IdentitiesOnly yes
diff --git a/docker/buildbot-worker-ntel/buildbot/.ssh/id.andrew.cooper.me b/docker/buildbot-worker-ntel/buildbot/.ssh/id.andrew.cooper.me
new file mode 100644
index 0000000..707f392
--- /dev/null
+++ b/docker/buildbot-worker-ntel/buildbot/.ssh/id.andrew.cooper.me
@@ -0,0 +1,30 @@
+-----BEGIN RSA PRIVATE KEY-----
+Proc-Type: 4,ENCRYPTED
+DEK-Info: AES-128-CBC,1281C9BA868D6806337DA195D4BF7AC4
+
+TwkensLykzo2j6obwvASPBg2hqMRSETTSN5MTg/FvPQmVCjmR2g5X/9CiJ2Sc1PV
+5EybLC45aMnckIGbSU0vZH4u5EGPtNDmyEroH+6CsaWWaB3jfgZoWldzZgq6dXQt
+wDHPJWqWzrDDoialwGkUt5lyc0k8BSoVxLcWTWJwUwv91LaojvKgv6e0Q9nFczSG
+SVICZp9dejCHFaGxli6ev9GUpG7aBktgfUkaAzZK6fNhGN/0MZHTSfT8A2/h0FDQ
+COsp/6uGk6wfuawMfCMBozoK/4FinHbvhpSNoCrdTTM9K9nWtQAclJYuYwDTsNc3
+83mKtXy0kNQxsB3Tq0KMALZpSX8SvSlU284fdMgwwXDsXXcw8emAk1dw6uKfBaQk
+viLwNGIozg/6M5MRI/N3d+e3lc+5v74/KwJf1Ark2JUOFteWjC5mLRT8nbRRC7Fg
+/s7awqqGc2zq0thQqQbPkMVuVlP9w97T6UZoyDhYN0rs2+2x/VMFs29NYBJeiMrN
+hqrqcn15RqzTO4F0oHSYyuvJErnXvTH2l7L1nhcYClAww8ZSw767PyqqK/zZmkNf
+PsnLEGmNEoJE6NJRPfuugSbb5dIV1UZag9wmEnBAQW0e3Kb2oRamyuXJF1VxQrks
+GJqugBBaIid/kF7R4B8uUlOYWC/iFS+04qukML/ljKqJWKMF8w56nXMmJrHyndZw
+S2s4PXRzMmFZU8JaELUyLRMT8yPhwcsfuIao1w3HExA7OIibZpEHIi7ZqJLkzqSv
+CfPGDq9RJR+RkFNOO6/YvOEXq7dprFjV6DARrEa6XhyYEK1QOK4MiS+6nMnU/TKb
+3qBrxVrCYIn3Q7fkyq9L5WGCipw9L5qauZw0ktdHWfN7HxMaKjdPVggQdmO96pS7
+esmBnAVL/gdKTlhSX/WKrjpxY92WY8SLcLfW8nE6nG06RfvSVizNi8lwGjNvbjLd
+IhOr7ETklHYfch/IaMVfU7Os5fwXaIM4GMQ27DmNMzCJJmIU8Pdhmp+mB4BL1SzC
+f7R2HuoPdbSGX7C/UYrupDC3lE3CU8BrezBJY86H12iCRNZM9g0ODXy9Zo9m2l4k
+mKZJWv/hB0O1CoHfNCv9+MEo61TiTPcFiIoozMpkL1cfbsmooD/09EzeL6VJhI3I
+OwiI5AZLwiF5Ol/JSAH8fEiV6yw/jmvjb15Qj/BKj+GLjKBnU5l+wZcrE7Wz00Cg
+pvFmJ0IffA/0Ws96UIPs4q3tkxNcg6Y+muTbN2Qk2+R8oab55UYIqQt1f4YNsj70
+p64U6V+oNnMuiPW84nvNq+MhD9Qjq4TYO6BNSPRAc8vs1uq7AymicUBSp8UZfvsx
+ZE6CJ862Sv+VsiUnr196mZS77zENZXd5XVLnsJKdBZM9A31zy5HD5HoiClmBvvn8
+ajDDsOwX3gSikA7+X8SmP7SSTDP1nnukOPCHz28k3YP7wY6X09FULMJBVgqWVOim
+17qqFdCYnn7fTWJZfHaRkR9+JILE5Ee0UsNvpaYcWV4ZygVR1IXxEwC+pOJ2FsRc
+b85mrCnYgC2TUDbUHpUw4YEIyUhuT4FlNlVyM/17hyz1Bv7gNLZ2h/xm98roDYAz
+-----END RSA PRIVATE KEY-----
diff --git a/docker/buildbot-worker-ntel/buildbot/.ssh/id.andrew.cooper.me.pub b/docker/buildbot-worker-ntel/buildbot/.ssh/id.andrew.cooper.me.pub
new file mode 100644
index 0000000..0ae10d7
--- /dev/null
+++ b/docker/buildbot-worker-ntel/buildbot/.ssh/id.andrew.cooper.me.pub
@@ -0,0 +1 @@
+ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQCejIRpFTpO1cP77JfTBHd56pdCyV8Ap1vbPABMz08UG7i/vifzHkCF+JX8mzxWsm1948ZKqXxB9iKDrUoSioZzBRe1PhxfZk0K9eoPOo+96gYUuAfEZ7hFIpwIpd/UozOzqaxH1XG77ougpn19ZzABihoYKXU8edmczsEAK8FNzWkl2254pWGxg1k+yxo3eHUY/I9+R0RiBoGZOl+eVpVUetxL+yWBPkP9ic6TsHoeOa6NAqoAFY+LsPrLwgRzNS+0rtIXnIfNdkxWEOur5sRQmt8yCaJCnEvWAZ7426xEB/SyPvNgrgQ3XTFfx8MFz+c82QxM/U4j0qb5B36F+Q3R buildbot@770dd006e82b
